src/QsCompiler/CompilationManager/CompilationManager.csproj

```
@@ -17,8 +17,7 @@
           https://docs.microsoft.com/en-us/dotnet/core/porting/third-party-deps#net-framework-compatibility-mode
       we disable this warning for the LanguageServer.Protocol package ONLY.
     -->
-    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="16.0.2264" NoWarn="NU1701" />
-    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.0.102" />
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="16.3.57" NoWarn="NU1701" />
```

Looks like 16.3.57 finally supports netstandard1.0, so warning NU1701 should no longer be raised on referencing the package.
```suggestion
    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="16.3.57" />
```

---

Cool!